runtime/Cpp/runtime/src/misc/Interval.h

```diff
@@ -26,59 +26,56 @@ namespace misc {
     ssize_t b;
 
     Interval();
-    explicit Interval(size_t a_, size_t b_); // For unsigned -> signed mappings.
```

Comment:
The `explicit` keyword's type conversion refers to conversion to the target class through a converting constructor, and whether that conversion can be implicit or must be explicit. I'm not sure how you are using this in the code. For example:

    explicit IntervalSet(size_t a, size_t b);

will prevent both:

    IntervalSet s = { 1, 2 };
    IntervalSet s = { -1, -2 }

and will allow both:

    IntervalSet s(1, 2);
    IntervalSet s(-1, -2);

What is the form you want to disallow?